components/booking/status.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react'
+import Button from '@mui/material/Button'
+import DescriptionOutlinedIcon from '@mui/icons-material/DescriptionOutlined'
+import Menu from '@mui/material/Menu'
+import MenuItem from '@mui/material/MenuItem'
+
+import { useState } from 'react'
+
+function BookingStatusDropdown({bookingId, bookingStatus, statusesTypes}) {
+  const [listMenu, setListMenu] = useState(null)
+  const [bookingStatusId, setBookingStatusId] = useState(bookingStatus)
+  const open = Boolean(listMenu)
+
+  function openStatusMenu(event: React.MouseEvent<HTMLButtonElement>) {
+    setListMenu(event.currentTarget)
+  }
+
+  function handleClose(statusId) {
```

Comment:
what happens if you click outside?

Reply:
the menu closes (that's why I also have "if" on the next line)